test/internal/AllowanceTest.cpp

```diff
@@ -0,0 +1,41 @@
+// Copyright 2004-present Facebook. All Rights Reserved.
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+#include "rsocket/internal/Allowance.h"
+
+using namespace ::testing;
+using namespace ::rsocket;
+
+TEST(AllowanceTest, Finite) {
+  Allowance sem;
```

Comment:
Maybe rename the `sem` variables in this file too.

Reply:
(y)